PyDSS/value_storage.py

```diff
@@ -321,12 +321,13 @@ class ValueContainer:
     _TYPE_MAPPING = {
         float: np.float,
         int: np.int,
-        complex: np.complex
+        complex: np.complex,
     }
 
     def __init__(self, value, hdf_store, path, max_size, max_chunk_bytes=None):
+
         dtype = self._TYPE_MAPPING.get(value.value_type)
-        assert dtype is not None
+        #assert dtype is not None
```

Comment:
I think this assert should be enabled.  If it's not, the code will fail with hard-to-interpret messages.  Do you have differing thoughts?

Reply:
I am fine with it. Uncommented